action_list:

  number_of_targets: 4

  action_0:
    action_type: pose
    translation: [0.4, 0.28, 0.2]
    orientation: [0.0, 0.0, 0.0]
    duration: 5.0

  action_1:
    action_type: pose
    translation: [0.4, 0.28, 0.08]
    orientation: [0.0, 0.0, 0.0]
    duration: 3.0

  action_2:
    action_type: pose
    translation: [ 0.4, -0.05, 0.08]
    orientation: [ 0.0, 0.0, 0.0]
    duration: 4.5

  action_3:
    action_type: pose
    translation: [ 0.4, -0.05, 0.2]
    orientation: [ 0.0, 0.0, 0.0]
    duration: 1.0
